packages/charts/react-charting/src/utilities/__snapshots__/UtilityUnitTests.test.ts.snap

```diff
@@ -3793,6 +3799,24 @@ Array [
 ]
 `;
 
+exports[`prepareDatapoints should return data points for a negative only range 1`] = `
```

Comment:
>exports[`prepareDatapoints should return data points for a negative only range 1`] [](http://example.com/codeflow?start=0&length=82)

could you explain this expected value. How is it coming to be this array?

Reply:
Rectified.